aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/data/AWSCognitoDeviceStore.kt

```diff
@@ -0,0 +1,39 @@
+package com.amplifyframework.auth.cognito.data
+
+import android.content.Context
+import com.amplifyframework.statemachine.codegen.data.DeviceMetadata
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class AWSCognitoDeviceStore(
```

Comment:
We should not create another store. We should instead add another object to the existing store.